aimmo-game/tests/test_simulation/test_map_generator.py

```diff
@@ -8,7 +8,7 @@
 from simulation.map_generator import get_random_edge_index
 from simulation.world_map import WorldMap
```

Comment:
I don't think you need this import anymore :)

Reply:
Indeed it is not.